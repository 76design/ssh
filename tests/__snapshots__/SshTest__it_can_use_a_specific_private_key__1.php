<?php return 'ssh -i /home/user/.ssh/id_rsa -o UserKnownHostsFile=/tmp/test user@example.com \'bash -se\' << \\EOF-SPATIE-SSH
whoami
EOF-SPATIE-SSH';
