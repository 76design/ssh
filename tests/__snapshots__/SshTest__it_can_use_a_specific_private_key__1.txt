ssh -i /home/user/.ssh/id_rsa -p  user@example.com 'bash -se' << \EOF-SPATIE-SSH
whoami
EOF-SPATIE-SSH